### PR TITLE
Implement "Show version info"

### LINKED ISSUE #74: Show version info
Is the command `v.ver` supposed to show the version of this extension or the version of V?

### PR DESCRIPTION
**Pull request by Acid147**
Closes #74.

### CODE REVIEW

File: src/client/commands.ts [hunk 1]
```diff
@@ -10,34 +12,43 @@ export function run() {
 /**
  * Build an optimized executable from current file.
  */
-export function prod() {}
+export function prod() { }
```
Comment (dhonx) [inline src/client/commands.ts]: Prettier will handle this. Did you run prettier.

Comment (Acid147) [reply src/client/commands.ts]: I used the wrong formatter.

Comment (dhonx) [reply src/client/commands.ts]: What formatter do you use?

Comment (Acid147) [reply src/client/commands.ts]: In this case the default one. I'm trying to figure out the right settings, so I can add a pull request that automatically sets prettier as the formatter.

**Comment by alexesprit** [comment]
Also, the the command description should be updated to something like that:
```
{
	"command": "v.ver",
	"title": "V: Show V version"
},
```
for better understanding what version this command displays.

**Pull request merged**